src/Hamiltonians/HubbardRealSpace.jl

```diff
@@ -0,0 +1,351 @@
+"""
+    real_space_interaction(::AbstractFockAddress, u)
+    real_space_interaction(::AbstractFockAddress, ::AbstractFockAddress, v)
+
+Return the real space interaction between one or two Fock states.
+"""
```

Comment:
```suggestion
"""
    local_interaction(::AbstractFockAddress, u)
    local_interaction(::SingleComponentFockAddress, ::SingleComponentFockAddress, v)

Return the sum of (mode-wise) local interactions ``\\frac{u}{2} \\sum_i n_i(n_i-1)`` of a single component
Fock state, or ``v \\sum_i n_{↑,i} n_{↓,i}`` between two Fock states. For a multi-component Fock state, 
return the eigenvalue of
```math
\\frac{1}{2}\\sum_{i, σ, τ} u_{σ,τ} a^†_{σ,i}a^†_{τ,i}a^†_{τ,i}a^†_{σ,i} ,
```
where `u::SMatrix` is a symmetric matrix of interaction constants, `i` is a mode index, 
and `σ, τ` are component indices.

See also [`BoseFS`](@ref), [`FermiFS`](@ref), [`CompositeFS`](@ref).
"""
```
So, I suggest to rename this function to `local_interaction()`. 
Please check the formula I wrote.

Sorry, somehow the md formatting got messed up.